tests/test_threadpoolctl.py

```diff
@@ -106,40 +112,6 @@ def test_set_threadpool_limits_bad_input():
         threadpool_limits(limits=(1, 2, 3))
 
 
-@pytest.mark.parametrize("user_api", (None, "blas", "openmp"))
-def test_thread_limit_context(user_api):
```

Comment:
It would be nice to test `threadpool_limits` without the context at some point, as it can be used as a function.

Reply:
Done in 5ec2a71.